content/docs/reference/public_private_keys/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: single
+
+title: 'Public and Private Keys' # The title (ON THE PAGE)
+lead: | # The lead below the title (ON THE PAGE)
+  <em>Providing access to information</em>
+
+
+description: | # SEO Description of the page (Shows in google and atsign.dev search)
+    Definition of public and private keys
+
+draft: false # Change this to "true" to hide the page
+toc: false # Change this to "true" to show the table of contents
```

Comment:
The other page has toc enabled, I would enable it here as well since this page seems longer.